### DISCUSSION TITLE
Cannot be trained on a huge dataset

### DISCUSSION THREAD

**Discussion by Romarin87** [deepmodeling/deepmd-kit#2386] [Q&A]
I wanted to train on a huge dataset, about 9.6 million points, 171 different systems. When I converted these systems to npy format for training, deepmd-kit was unable to train, and when I randomly trained a subset of these systems, it was normal to train.
What should I do to train properly?

**Comment by njzjz** [comment]
To my experience, the only issue with the huge data is that it takes a lot of time to start up.
